include/EVT/io/I2C.hpp

```diff
@@ -97,6 +97,48 @@ class I2C {
     void readReg(uint8_t addr, uint8_t* reg, uint8_t regLength,
             uint8_t* bytes, uint8_t length);
 
+    /**
+     * Write a single byte to a register in memory.
```

Comment:
I may add to the comment a little more clarification (or a link to some explanation) that memory modules follow a slightly different I2C pattern with the reads and writes.